### ISSUE TITLE
Missing accessibility lable

### ISSUE THREAD

**Issue by TStArK1233** [SimpleMobileTools/Simple-Notes#632]
![IMG_20230517_200854](https://github.com/SimpleMobileTools/Simple-Notes/assets/133884373/6d989a49-848d-4694-ad0a-ed3f11b03865)

The button marked on red are not labelled. Thank You for working on it😊😊.

**Comment by Aga-C** [comment]
This also applies to Simple File Manager.

**Comment by tibbi** [comment]
will be fixed in 6.15.6
